RTN-008.tex

```diff
@@ -62,114 +63,113 @@
-
-In \citeds{DMTN-107} the DM-SST suggests that incremental template generation in the first year of LSST operations provides the best balance of early template availability and artifact rejection.
-In this scenario new templates are regularly generated throughout the year whenever sufficient imaging becomes available in sky areas and filters without templates.
-Having preexisting templates will allow standard alert processing in GW TOO fields, easing the operational impacts of TOO observations.
+Timely generation of ``incremental'' templates as commissioning and early operations proceed will thus improve the likelihood that a TOO could be processed with standard Alert Produciton.
+%In \citeds{DMTN-107} the DM-SST suggests that incremental template generation in the first year of LSST operations provides the best balance of early template availability and artifact rejection.
+In this scenario new templates are regularly generated throughout the year whenever sufficient imaging becomes available in sky areas and filters without templates \citedsp{RTN-011}.
+%Having preexisting templates will allow standard alert processing in GW TOO fields, easing the operational impacts of TOO observations.
 
 Additionally, evidence of variability prior to the GW trigger provides one of the most effective ways to rule out unassociated sources in the GW localization region \citep[e.g.,][]{2019GCN.24223....1C, 2019GCN.26430....1S}.
 At LSST depths there will be many unrelated events, and little recent prior imaging of sufficient depth from other surveys that can constrain the explosion date.
-Already today at DECam followup depths the number of unrelated events is straining the ability of the worldwide community to obtain classification spectra \citep[e.g.,][]{2019ApJ...881L..16A}.
 
 In the steady state survey, LSST's own history will provide substantial filtering: simply selecting \DIAObjects created after the the trigger will exclude most unrelated contaminants.
 In early operations, however, this history will not be available, so not only distant supernovae but also variable stars and AGN will confound efforts to identify plausible GW counterparts.
 
-For this reason it will be very helpful for LSST to have reliable standard image differencing processing---and hence history---as soon as is practical, providing further impetus for incremental template building.
+For this reason it will be very helpful for Rubin to have reliable standard image differencing processing---and hence history---as soon as is practical, providing further impetus for incremental template building.
 While custom processing and association could be used to derive this history (\S \ref{sec:processing}) it would require substantial additional effort relative to LSST's standard processing.
```

Comment:
I agree with your reasoning on this and on the value to the science. TOO is a special program. Our construction requirements state that SPs can trigger their own data processing recipes during nightly AP but that new algorithm development or allocation of excessive resources is out of scope. Custom processing and association seem to fall into these categories, dnd hence is an upscope.  i wonder if we can look to in-kind contributions for help?

Reply:
I think SPs triggering data processing recipes is a red herring here--if we could do any kind of automated processing, we would do so.  The problem is that we don't have the images to support it.  So we're not asking DM to construct any new pipelines: we're asking the commissioning and/or operations personnel to do some manual analysis of the data.  That's work that's certainly not been clearly scoped for those teams--but it's not a _DM construction_ upscope, in my view.  We're not adding any requirements.

Reply:
Discussed off line - Eric will rephrase

Reply:
I think I follow. Given the need to hold data close over a timescale that surpasses GW follow needs, Commissioning and Ops teams will need to do something. FYI we will be having a post PCW workshop with the agencies on data security. So hold times and when they are in effect will be discussed. We think the SV surveys at the end of commissioning can be treated like ops, not commissioning. So 80 hr, not 30 days.

Reply:
Great, that's useful to know.  I'm going to leave the text as is here until we get confirmation on the SV surveys.